Gemfile

```diff
@@ -23,6 +23,7 @@ gem 'uglifier', '>= 1.3.0'
 gem 'coffee-rails', '~> 4.2'
 # See https://github.com/rails/execjs#readme for more supported runtimes
 # gem 'therubyracer', platforms: :ruby
+gem 'progressbar'
```

Comment:
When you run `rails db:seed` it will show a progress bar across your terminal as it creates new objects in the database.  The whole process takes... maybe 20-30 seconds?